### ISSUE TITLE
Amulet pal file added in PR #3078 breaks git on mac

### ISSUE THREAD

**Issue by grunt-lucas** [rh-hideout/pokeemerald-expansion#3085]
### Description

I tested on two different mac laptops. Clone a clean pokeemerald repo. Run `git checkout upcoming`. You will have an unclean index with the following change:
```
On branch upcoming
Your branch is up to date with 'origin/upcoming'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   graphics/items/icon_palettes/clear_amulet.pal

no changes added to commit (use "git add" and/or "git commit -a")
```
It is impossible to clear this change. You can't `stash` it. You can't `restore` it. Git will not allow you to checkout any other branches either since there are local changes present.

I am not sure what is going on. I thought it might be a platform-specific line endings thing, but I have never had this error with other pal files. And I see the expansion has `*.pal text eol=crlf` defined in `.gitattributes` so that should be handling things.

### Version

upcoming (Edge)

### Upcoming Version

a44af6f06

### Discord contact info

lucas7562

**Comment by AsparagusEduardo** [comment]
I think it was because it was added via GitHub upload. I found something similar happening with the issue template files.

**Comment by AsparagusEduardo** [comment]
It does happen in WSL as well btw. Please let me know if this commit fixes the issue:
https://github.com/AsparagusEduardo/pokeemerald/commit/9db1230944461b27c775ed7f32fab89e43908104

**Comment by grunt-lucas** [comment]
I wanted to note for anyone who can't get rid of the change in their staging area:
```
git rm --cached -r .
git reset --hard
```
should revert your staging area and clear the changes.

**Comment by grunt-lucas** [comment]
Fixed by PR #3806
